osrdyne/src/config.rs

```diff
@@ -32,6 +34,22 @@ pub struct OsrdyneConfig {
     pub max_length_bytes: Option<usize>,
     pub api_address: String,
     pub extra_lifetime: Option<Duration>,
+    pub opentelemetry: Option<OpentelemetryConfig>,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct OpentelemetryConfig {
+    pub service_name: Option<String>,
+    pub endpoint: Url,
+}
+
+impl Default for OpentelemetryConfig {
+    fn default() -> Self {
+        Self {
+            service_name: None,
```

Comment:
Why making it optional? `"osrdyne".into()` was a good default, don't you think?

Reply:
Nope, was still messing when set from environment variables